yarn.lock

```diff
@@ -7875,21 +7864,21 @@ aws-appsync@^4.1.1, aws-appsync@^4.1.9:
     url "^0.11.0"
     uuid "3.x"
 
-aws-cdk-lib@~2.53.0:
-  version "2.53.0"
-  resolved "https://registry.npmjs.org/aws-cdk-lib/-/aws-cdk-lib-2.53.0.tgz#c36ecfb87ba4e7d1cdfe0c620161beda0ce4aa67"
-  integrity sha512-ADpJ9luJxzmOrP/GJ37nA1YtdNmsOsjE+NZnQOpB7YL0spUxZJGNVAselaFLSDZmIsC6d9mSnW81fj4SDP1gAw==
+aws-cdk-lib@~2.53.0, aws-cdk-lib@~2.68.0:
```

Comment:
we'll probably need another update to this file once the CLI packages like `category-custom` and `cloudformation-provider` get updated to this higher version. That is where this `~2.53.0` is coming from.

Reply:
Correct, but we want your tests run with uniform cdk.